data/pgroonga.sql

```diff
@@ -437,45 +437,19 @@ CREATE FUNCTION pgroonga_wal_set_applied_position()
 	LANGUAGE C
 	STRICT;
 
-DO LANGUAGE plpgsql $$
-DECLARE
-	rec record;
-BEGIN
-	-- A column in pg_stat_wal_receiver is different for 12 and 13 or higher.
-	--* received_lsn: 12
-	--* flushed_lsn: 13 or higher
-	SELECT * INTO STRICT rec from pg_settings WHERE name = 'server_version';
-	IF SUBSTR(rec.setting, 1, 2) >= '13' THEN
-		CREATE FUNCTION pgroonga_list_lagged_indexes()
-			RETURNS SETOF text AS '
-				SELECT name FROM pgroonga_wal_status()
-				WHERE current_block != last_block
-					OR current_offset != current_offset
-					OR current_size != last_size
-					OR EXISTS(
-						SELECT 1 FROM pg_stat_wal_receiver
-						WHERE flushed_lsn != latest_end_lsn
-					);
-			' LANGUAGE SQL
-			STRICT
-			PARALLEL SAFE;
-	ELSE
-		CREATE FUNCTION pgroonga_list_lagged_indexes()
-			RETURNS SETOF text AS '
-				SELECT name FROM pgroonga_wal_status()
-				WHERE current_block != last_block
-					OR current_offset != current_offset
-					OR current_size != last_size
-					OR EXISTS(
-						SELECT 1 FROM pg_stat_wal_receiver
-						WHERE received_lsn != latest_end_lsn
-					);
-			' LANGUAGE SQL
-			STRICT
-			PARALLEL SAFE;
-	END IF;
-END;
-$$;
+CREATE FUNCTION pgroonga_list_lagged_indexes()
+	RETURNS SETOF text AS '
+		SELECT name FROM pgroonga_wal_status()
+		WHERE current_block != last_block
+			OR current_offset != current_offset
```

Comment:
Is using `last_offset` correct here because the condition was always true ?
```suggestion
			OR current_offset != last_offset
```

Reply:
@abetomo ↑

Reply:
良いところに気づきましたね！
`current_offset != last_offset` が合ってます。

Reply:
fix: [fa8a02c](https://github.com/pgroonga/pgroonga/pull/615/commits/fa8a02c8c896954f2a4585f1ece90707a73e7237) Yatta!

Reply:
アップグレードSQLとかで直すとかもしないといけないので、ここで直すのはやめて、別PRにしませんか？

Reply:
なるほどです！この変更は切り出して別で対応しようと思います。

Reply:
fix: [c9bf3a6](https://github.com/pgroonga/pgroonga/pull/615/commits/c9bf3a674d624ff32fc083ebc4267b90543a6e7f)